src/Mvc/QtView.php

```diff
@@ -173,7 +179,13 @@ public function render(string $view, array $params = []): ?string
             AssetManager::getInstance()->register($this->assets);
         }
 
-        return $this->renderFile($this->layout);
+		$content = $this->renderFile($this->layout);
+
+	    if (route_cache()){
+		    (new ViewCache())->set(route_uri(), $content, session()->getId());
+	    }
+
+        return $content;
```

Comment:
If route is cacheable and also minifiable, this return still returns normal rendered view